docs/components/movement-sensor/imu/imu-wit-hwt905.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Configure a WitMotion HWT905-TTL IMU"
+linkTitle: "imu-wit-hwt905"
+weight: 10
+type: "docs"
+description: "Configure a WitMotion HWT905-TTL IMU on your machine. Once configured, use the API to obtain the AngularVelocity, Orientation, CompassHeading and LinearAcceleration."
+images: ["/icons/components/imu.svg"]
+# SMEs: Bucket Team
```

Comment:
This is true, but specifically on the bucket team, Susmita knows the most. Worth mentioning here? I can go either way.

Reply:
```suggestion
# SMEs: Susmita, Bucket Team
```